Dockerfile.assisted-installer

```diff
@@ -1,8 +1,16 @@
-FROM quay.io/centos/centos:centos8
-copy build/installer /
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
```

Comment:
I see we're using mostly `registry.svc.ci.openshift.org/openshift/release:golang-1.15` on other places. Is there a difference?

On [assisted-service](https://github.com/openshift/assisted-service/blob/33090c305040b9202c7f84b4be9489568e79c84b/Dockerfile.assisted-service#L18-L24) for example. But then we empty `GOFLAGS`

Reply:
I think this was a copy/paste error on my part. Will go back to `registry.svc.ci.openshift.org/openshift/release:golang-1.15` or have an answer as to why ;)